specs/bundling.md

```diff
@@ -0,0 +1,27 @@
+## Safe Module Merkle Tree Bundling Logic
+
+### Goal
+A function that transforms an arbitrary set of transactions on an arbitrary set of networks into a Merkle Tree which is executable by the SphinxModule. See the SphinxModule Merkle Tree spec for more detail on the exact structure and content of the Merkle tree and its leafs.
+
+### Secondary Goals
+- Simple and auditable
+- Should be a function that is agnostic to the source of transactions so it can be used to assemble a Merkle Tree based on transactions generated from any scripting framework (i.e Foundry, Hardhat Ignition, some arbitrary future framework)
+
+### Input
+Accepts an object where the keys are canonical chain ids and the values are deployment data objects.
+
+#### Deployment Object Fields:
+- nonce: The current nonce of the Safe on the target network
+- txs: An array of Sphinx Transaction objects
+
+#### Sphinx Transaction Object Fields:
```

Comment:
Would be helpful to have the actual TypeScript names of these data types. Another option is to just document these variable types in TypeScript then link to them in a 'Relevant Files' section like we do in the other specs. That may be preferable b/c we'll probably want to document the TypeScript logic, and this way we don't need to duplicate it